.travis.yml

```diff
@@ -1,5 +1,8 @@
 # make it explicit that we favor the new container-based travis workers
-sudo: false
+sudo: required
```

Comment:
I know this is necessary, but this will negatively impact all the other builds, won't it?

I wonder if there is a way to conditionally enable docker service and sudo only when it's needed. Another alternative is to use this docker image in CircleCI but it doesn't have a cron functionality..

Reply:
It is possible to use `sudo: required` only in one job. See https://alexwlchan.net/2018/08/selective-sudo-on-travis/

The same might apply to the `- services: docker` entry.

Reply:
ill look some time tomorrow. one curious question, why using circleCI in some cases and travis in the other?

Reply:
FWIW, Travis deprecated the container-based infrastructure several months ago, and you’re probably running with sudo-enabled builds already. So this may not make a difference.

See:

* https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures
* https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517
* https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments

(Also, if you do end up changing this, you should change the comment in line 1 as well!)

Reply:
sounds good, nice to know! added a note and removed the `sudo: required` statement